docs/directory-structure.md

```
@@ -0,0 +1,86 @@
+# 디렉토리 구조 가이드
+
+## 전체 패키지 구조
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)